### PR TITLE
Dev example http

### PR DESCRIPTION
**Pull request by otaviolcs3**

### CODE REVIEW

**Review by willallves — APPROVED** [review]

File: examples/README.md [hunk 1]
```diff
@@ -6,4 +6,6 @@ This section contains a set of standalone examples that demonstrate different us
 
 - [Mutually Authenticated TLS (mTLS)](spiffe-tls/README.md): _Establish mTLS connections between workloads using automatically rotated X.509 SVIDs obtained from the SPIFFE Workload API._
 
-- [HTTP over TLS with JWT and X.509 SVIDs](spiffe-jwt/README.md): _Send HTTP requests between workload over a TLS + JWT authentication using automatically rotated X.509 SVIDs and JWT SVIDs from the SPIFFE Workload API._
\ No newline at end of file
+- [HTTP over TLS with JWT and X.509 SVIDs](spiffe-jwt/README.md): _Send HTTP requests between workload over a TLS + JWT authentication using automatically rotated X.509 SVIDs and JWT SVIDs from the SPIFFE Workload API._
```
Comment (arianalima) [inline examples/README.md:9]:
```suggestion
- [HTTP over TLS with JWT and X.509 SVIDs](spiffe-jwt/README.md): _Send HTTP requests between workloads over a TLS + JWT authentication using automatically rotated X.509 SVIDs and JWT SVIDs from the SPIFFE Workload API._
```

File: examples/spiffe-http/README.md [hunk 2]
```diff
@@ -0,0 +1,111 @@
...
+- The trust domain is `example.org`
+- The agent SPIFFE ID is `spiffe://example.org/host`.
+- There is a `server-workload` and `client-workload` user in the system.
+
+### 1. Create the registration entries
+Create the registration entries for the client and server workloads:
+
+Server:
+```bash
+./spire-server entry create -spiffeID spiffe://example.org/server \
```
Comment (arianalima) [inline examples/spiffe-http/README.md:81]:
```suggestion
spire-server entry create -spiffeID spiffe://example.org/server \
```

File: examples/spiffe-http/README.md [hunk 3]
```diff
@@ -0,0 +1,111 @@
...
+                            -parentID spiffe://example.org/host \
+                            -selector unix:user:server-workload
+```
+
+Client: 
+```bash
+./spire-server entry create -spiffeID spiffe://example.org/client \
```
Comment (arianalima) [inline examples/spiffe-http/README.md:88]:
```suggestion
spire-server entry create -spiffeID spiffe://example.org/client \
```

**Review by arianalima — CHANGES_REQUESTED** [review]
Couldn't run the example following the README steps

File: examples/spiffe-http/README.md [hunk 4]
```diff
@@ -0,0 +1,111 @@
...
+Client: 
+```bash
+spire-server entry create -spiffeID spiffe://example.org/client \
+                            -parentID spiffe://example.org/host \
+                            -selector unix:user:client-workload
+```
+
+### 2. Start the server
+```bash
+./spiffe_http_server
```
Comment (arianalima) [inline examples/spiffe-http/README.md:95]:
```suggestion
su - server-workload -c "./spiffe_http_server"
```

File: examples/spiffe-http/README.md [hunk 5]
```diff
@@ -0,0 +1,111 @@
...
+```
+
+### 3. Run the client
+```bash
+./spiffe_http_client
```
Comment (arianalima) [inline examples/spiffe-http/README.md:100]:
```suggestion
su - client-workload -c "./spiffe_http_client"
```

File: examples/spiffe-http/README.md (hunk 4 above)
Comment (arianalima) [reply examples/spiffe-http/README.md:95]: Also, when running it without "server-workload" user we get a segfault:
```
res: 7
curl_easy_perform() failed: Couldn't connect to server
Segmentation fault (core dumped)
```

**Review by tjamir — APPROVED** [review]

**Pull request merged**